src/Simplex/Chat.hs

```diff
@@ -785,6 +790,19 @@ processChatCommand' vr = \case
       unzipMaybe3 :: Maybe (a, b, c) -> (Maybe a, Maybe b, Maybe c)
       unzipMaybe3 (Just (a, b, c)) = (Just a, Just b, Just c)
       unzipMaybe3 _ = (Nothing, Nothing, Nothing)
+  APICreateChatItem folderId (ComposedMessage file_ quotedItemId_ mc) -> withUser $ \user -> do
+    forM_ quotedItemId_ $ \_ -> throwError $ ChatError $ CECommandError "not supported"
+    nf <- withStore $ \db -> getNoteFolder db user folderId
+    -- TODO: assertLocalAllowed user MDSnd nf XMsgNew_
+    ci'@ChatItem {meta = CIMeta {itemId, itemTs}} <- createLocalChatItem user (CDLocalSnd nf) (CISndMsgContent mc) Nothing
+    ciFile_ <- forM file_ $ \cf@CryptoFile {filePath, cryptoArgs} -> do
+      fsFilePath <- toFSFilePath filePath -- XXX: only used for size?..
```

Comment:
it also appends files folder (it changes in mobile apps between updates/installations so we have to work with relative paths)
```suggestion
      fsFilePath <- toFSFilePath filePath
```

Reply:
I think it's because chat has no reason to read its contents, but size is used for metadata (e.g. sent in invitation, or in this case only displayed on chat item vi CIFile)